decomposed/extensions/MDDE_LDM/Profile/Model/Methods/mdde_InitializeExportExtension.xml

```diff
@@ -32,57 +51,80 @@ Function GTLOutputHasErrors(GTLResult)
    End If
 End Function
 
-Sub processPackage(objModel, objPackage, attributeTemplateName, attributeExtendedAttributeName)
+Function processPackage(objModel, objPackage, attributeTemplateName, attributeExtendedAttributeName)
 
    &#39;Loop over Entities
-   Dim ent
+   Dim ent, success1, success2
+   processPackage = True
+   success1 = False
```

Comment:
Can you give success1 and success2 a bit more meaningful name?

Reply:
yes